src/services/users.ts

```diff
@@ -56,26 +56,52 @@ const removeUser = async function (userId: string, password: string) {
 
   if (passwordsMatch) {
     // add email check once login is implemented
-    try {
-      const deleteUser = 'DELETE FROM users WHERE id = %L';
-      const result = await dbQuery(deleteUser, userId);
-
-      if (result.rowCount > 0) {
-        return { message: 'Your account has been deleted' };
-      }
-    } catch (error) {
-      return { message: 'Something went wrong.' };
+    // try {
+    //   const deleteUser = 'DELETE FROM users WHERE id = %L';
+    //   const result = await dbQuery(deleteUser, userId);
+
+    //   if (result.rowCount > 0) {
+    //     return { message: 'Your account has been deleted' };
+    //   }
+    // } catch (error) {
+    //   return { message: 'Something went wrong.' };
+    // }
+
+    const deleteUser = 'DELETE FROM users WHERE id = %L';
```

Comment:
Use `RETURNING *` at the end of the request and then return the deleted user (if successful) or `throw boom` if unsuccessful? You could then, for example, use the user data in the front-end for the final farewell: "User xxx with email yyy has been deleted.".